a10_octavia/controller/worker/controller_worker.py

```diff
@@ -208,7 +205,7 @@ def create_listener(self, listener_id):
 
         load_balancer = listener.load_balancer
 
-        if listener.project_id in self.rack_dict:
+        if listener.project_id in CONF.RACK_VTHUNDER.devices:
```

Comment:
`RACK_VTHUNDER` capitalizing this is improper. We should keep consistent and use lower case with underscore separators.